multiraft/rpc.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tamir Duberstein (tamird@gmail.com)
+
+syntax = "proto2";
+package cockroach.multiraft;
+option go_package = "multiraft";
+
+import "github.com/coreos/etcd/raft/raftpb/raft.proto";
```

Comment:
I agree that we should fix etcd's build system and then remove $GOPATH/src from our import path. I don't think that should hold up this change, though.